### PR TITLE
Merge cloud authentication portal

### PR DESCRIPTION
**Pull request by pavinduLakshan**
### Purpose

$subject

### Related Issues
- https://github.com/wso2/product-is/issues/16416

### Related PRs
- Related PR `#1` or (None)

### Checklist
- [ ] e2e cypress tests locally verified.
- [ ] Manual test round performed and verified.
- [ ] UX/UI review done on the final implementation.
- [ ] Documentation provided. (Add links if there are any)
- [ ] Unit tests provided. (Add links if there are any)
- [ ] Integration tests provided. (Add links if there are any)

### Security checks
- [x] Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines?
- [ ] Ran FindSecurityBugs plugin and verified report?
- [x] Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets?

### CODE REVIEW

**Review by DonOmalVindula — APPROVED** [review]

File: java/apps/authentication-portal/src/main/webapp/add-security-questions.jsp [hunk 1]
```diff
@@ -97,18 +93,16 @@
 </head>
 <body class="login-portal layout authentication-portal-layout">
     <layout:main layoutName="<%= layout %>" layoutFileRelativePath="<%= layoutFileRelativePath %>" data="<%= layoutData %>" >
-        <layout:component componentName="ProductHeader">
+        <layout:component componentName="ProductHeader" >
```
Comment (DonOmalVindula) [inline java/apps/authentication-portal/src/main/webapp/add-security-questions.jsp:96]: Remove whitespace. Check in other places as well

File: java/apps/authentication-portal/src/main/webapp/authenticate.jsp [hunk 2]
```diff
@@ -156,18 +150,16 @@
                 </form>
             </div>
         </layout:component>
-        <layout:component componentName="ProductFooter">
+        <layout:component componentName="ProductFooter" >
```
Comment (DonOmalVindula) [inline java/apps/authentication-portal/src/main/webapp/authenticate.jsp:153]: Remove whitespace. Check in other places as well

**Review by DonOmalVindula — APPROVED** [review]

File: java/apps/authentication-portal/src/main/webapp/add-security-questions.jsp (hunk 1 above)
Comment (pavinduLakshan) [reply java/apps/authentication-portal/src/main/webapp/add-security-questions.jsp]: Fixed with e89ab17

File: java/apps/authentication-portal/src/main/webapp/authenticate.jsp (hunk 2 above)
Comment (pavinduLakshan) [reply java/apps/authentication-portal/src/main/webapp/authenticate.jsp]: Fixed with e89ab17

File: java/apps/authentication-portal/src/main/webapp/authenticate.jsp [hunk 3]
```diff
@@ -1,12 +1,12 @@
 <%--
-  ~ Copyright (c) 2018-2023, WSO2 LLC. (https://www.wso2.com).
+  ~ Copyright (c) 2018-2023, WSO2 LLC. (http://www.wso2.com).
```
[the remark this answers is not in the document]
Comment (pavinduLakshan) [reply java/apps/authentication-portal/src/main/webapp/authenticate.jsp]: Fixed with 61feb35

**Pull request merged**